client/components/mma/billing/Billing.tsx

```diff
@@ -306,7 +305,31 @@ function renderProductBillingInfo([mmaCategory, productDetails]: [
 }
 
 function renderInAppPurchase(value: AppSubscription) {
-	return <div key={value.subscriptionId}>todo</div>;
+	return (
+		<div css={subHeadingBorderTopCss} key={value.subscriptionId}>
+			<h2
+				css={css`
+					${subHeadingTitleCss}
+					margin: 0;
+				`}
+			>
+				App Subscription
+			</h2>
+			<div
+				css={css`
+					${textSans.medium()};
+					border: 1px solid ${neutral[20]};
+					display: flex;
+					flex-wrap: wrap;
```

Comment:
It doesn't look like this needs the flex properties as it only has text content?

Reply:
True, was copied from the other billing tables